test/rpc-integration/test-subsorts/state-not-subsort.execute

```diff
@@ -60,6 +60,22 @@
               }
             ]
           },
+          {
+            "tag": "App",
+            "name": "Lbl'-LT-'int'-GT-'",
+            "sorts": [],
+            "args": [
+              {
+                "tag": "EVar",
+                "name": "X",
+                "sort": {
+                  "tag": "SortApp",
+                  "name": "SortInt",
+                  "args": []
+                }
+              }
+            ]
+          },
```

Comment:
Interesting... 
The [configuration in `diamond/test.k`](https://github.com/runtimeverification/hs-backend-booster/blob/53a66a27638834939612e540564f63be90d13079/test/rpc-integration/resources/diamond/test.k) (which is used for this test) is `<k> $PGM:State ~> .k </k><int> 42 </int>, therefore `generatedTop` takes 3 arguments.
So this input actually has _two_ sort errors now (wrong injection within `isInt` in the K-cell, wrong argument sort here), but only the first one is reported (expected response "`KItem` is not subsort of `K`")
Maybe we should 
* separately test the different error cases (wrong injection, wrong argument count, wrong argument sort)
* use `subsort.k` from the internalisation tests for this rpc test? (but probably not necessary)

Reply:
I've added two new tests to exercise each of the new checks introduced